admin_manual/configuration_server/occ_command.rst

```diff
@@ -991,6 +992,35 @@ command after modifying ``config/mimetypemapping.json``. If you change a
 mimetype, run ``maintenance:mimetype:update-db --repair-filecache`` to apply the 
 change to existing files.
 
+.. _reports_commands_label:
+   
+Reports
+-------
+
+If you're working with ownCloud support and need to send them a configuration summary, you can generate it using the ``configreport:generate`` command. 
+This command generates the same JSON-based report as the Admin Config Report, which you can access under ``admin -> Settings -> Admin -> Help & Tips -> Download ownCloud config report``.
+
+From the command-line in the root directory of your ownCloud installation, run it as your webserver user as follows, (assuming your webserver user is www-data):
+
+::
+
+  sudo -u www-data occ configreport:generate
```

Comment:
Should´t it be ``sudo -u www-data ./occ configreport:generate``  (occ -> ./occ)
Below the same?